src/wasm-exts/ast.ml

```diff
@@ -110,6 +110,10 @@ and instr' =
   | Store of storeop                  (* write memory at address *)
   | MemorySize                        (* size of linear memory *)
   | MemoryGrow                        (* grow linear memory *)
+  | StableSize                        (* size of stable memory *)
```

Comment:
Maybe a short comment could be helpful, indicating that these are custom instructions (different to Wasm spec/proposal).